apps/dg/components/case_card/case_card_model.js

```diff
@@ -25,4 +25,11 @@ sc_require('components/case_table/data_context_base_model');
  @extends DG.DataContextBaseModel
  */
 
-DG.CaseCardModel = DG.DataContextBaseModel.extend();
+DG.CaseCardModel = DG.DataContextBaseModel.extend({
+    /**
+     * Percentage width of attribute name column
+     * 
+     * @property {number}
+     */
+    columnWidthPct: null
```

Comment:
This is the ultimate source of truth. It is passed down to the React components that need it as props and callbacks are provided for setting it when resizing occurs.